### DISCUSSION TITLE
bcprov-jdk15on - CVE 2023 33201

### DISCUSSION THREAD

**Discussion by OscarFolhaMsgLifeIberia** [bcgit/bc-java#1435] [General]
I know you guys already fixed this in version bc-java 1.74.0 but here with bcprov-jdk15on the latest version does not have this fixed.

![image](https://github.com/bcgit/bc-java/assets/118288240/884a8a6c-8a0b-42a9-a7f2-95eeedb01532)

Do you know a way around this?

**Comment by dghgit** [comment]
bcprov-jdk15on was split into bcprov-jdk15to18 and bcprov-jdk18on - I'd recommend updating to whichever of those best suits your purpose.
